core/math/vector2.cpp

```diff
@@ -34,6 +34,10 @@ real_t Vector2::angle() const {
 	return Math::atan2(y, x);
 }
 
+Vector2 Vector2::from_angle(real_t p_angle) {
```

Comment:
Maybe:

```suggestion
Vector2 Vector2::from_angle(real_t p_radians) {
```

since `from_angle` already implies it,